### DISCUSSION TITLE
Dev related Discussion

### DISCUSSION THREAD

**Discussion by orama254** [reactdeveloperske/reactdevske-website#12] [Ideas]
## 💡 Welcome to the Dev related Discussion!
  We’re using this Dev related Discussions as a place to  share Dev suggestions, architecture, implementation and refactoring. We hope that you:
  * Share dev ideas.
  * showcase some development methodologies
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with a piece of development approach.

**Comment by orama254** [comment]
Initiating a discussion on folder structure for the current project...We had some really good points raised in the standup call on the 16th of June 2022.

We currently don't have a preferred folder structure for the project so your suggestions are welcome

**Reply by orama254** [reply]
Added this thread to a new discussion [HERE](https://github.com/reactdeveloperske/reactdevske-website/discussions/61)

**Comment by lekipising** [comment]
Hey @orama254 
I had some questions about the designs.

I noted that between desktop and mobile, we have different font families being used. Is that intentional? 

I was checking the Mission Pillars section.

**Comment by nyandika** [comment]
@orama254 I noticed the tests have been failing. I checked them out locally and figured it is probably the ignored .env file containing keys and such. Have you added the keys to github secrets so the action can read them as environment objects?

**Reply by orama254** [reply]
Hello @nyandika, appreciate the heads up. 

Will try make the changes as suggested then get back to you incase anything comes up

**Reply by orama254** [reply]
Hello @nyandika 

I realized that the tests keep failing with webkit locally and all of them fail on github actions.

I also tried updating the e2e workflow based on the updated workflows on playwright [Here](https://playwright.dev/docs/ci-intro#github-actions) and also [this thread here](https://github.com/microsoft/playwright/issues/20664) which is still failing.

Kindly have a look at the [latest commits](https://github.com/reactdeveloperske/reactdevske-website/pull/131/commits) on Pull #131 for reference and further guidance.
